templates/plans/purchase.hamlet

```diff
@@ -1,19 +1,23 @@
 $with price <- planPrice plan
```

Comment:
Looks like `price` isn't used in the `$nothing` branch here, so you could move `$with price` inside the `$maybe`.